src/advanced/consensus/requests.md

```diff
@@ -23,20 +24,22 @@ an opportunity to learn certain information about the state of the message
 author (a node that has signed the message; the message author may differ from
 the peer that the message recipient got the message from), if the author is not
 Byzantine. The receiving node saves this information in
-[the RequestState structure](#sending-requests).
+[the StateRequest structure](#sending-requests).
```

Comment:
I was hesitating on that for a while, but at last it seemed to me it would be better from the point of view of unity of terminology. Your explanation, of course, denounces my assumption.